lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -130,6 +133,76 @@ class ReportUIFeatures {
     dropdown.addEventListener('click', this.onExport);
   }
 
+  _setupThirdPartyFilter() {
+    // get all tables with a text url
+    /** @type {Array<HTMLTableElement>} */
+    const tables = Array.from(this._document.querySelectorAll('.lh-table'));
+    const thirdPartyFilterAuditExclusions = tables
```

Comment:
heh. i bet this didn't make sense... what i meant to say was.. lets try an array for these ids.. i was thinking like this:

```js
const thirdPartyFilterAuditExclusions = ['uses-rel-preconnect'];
const tablesWithUrls = tables      
      .filter(el => el.querySelector('td.lh-table-column--url'))
      .filter(el => !thirdPartyFilterAuditExclusions.includes( /** @type {Element} */ (el.closest('.lh-audit')).id));
```